no-k8s/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM registry.gitlab.factory.social.gouv.fr/socialgouv/docker/kosko:4.2.0
+
+RUN apk add --update --no-cache git=~2
+
+# renovate: datasource=github-tags lookupName=socialgouv/.k8s versioning=1.0.1
+RUN git clone --depth 1 --branch v1.0.1 https://github.com/SocialGouv/.k8s /.k8s
+
+RUN cd .k8s; yarn install --frozen-lockfile; cd -
```

We might want to have a production ready version with no extra cache by default

```suggestion
WORKDIR /.k8s
RUN yarn install --frozen-lockfile --production --cache-folder /dev/shm/yarn
```

---

Done.